### PR TITLE
[ET-767] Improve a11y on tickets & rsvp forms

### PR DESCRIPTION
**Pull request by juanfra**
### 🎫 Ticket

[ET-767] 

### 🗒️ Description

- Adding label to the quantity input to improve accessibility in the tickets and RSVP forms.

### 🎥 Artifacts 

https://www.loom.com/share/47433e15a6e04b7d9b94df9c99b889d4

### ✔️ Checklist
- [x] I've included a readme.txt Changelog entry. 
- [ ] My code is tested. 
- [ ] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ET-767]: https://theeventscalendar.atlassian.net/browse/ET-767

### CODE REVIEW

**Comment by juanfra** [comment]
Updating snapshots. Docker is killing my machine.

File: src/views/blocks/rsvp/form/quantity-input.php [hunk 1]
```diff
@@ -34,8 +35,15 @@
 
 $max_at_a_time = $tickets_handler->get_ticket_max_purchase( $ticket->ID );
 ?>
+<label
+	class="screen-reader-text"
+	for="quantity_<?php echo absint( $ticket->ID ); ?>"
```
Comment (paulmskim) [inline src/views/blocks/rsvp/form/quantity-input.php:40]: esc_attr() here and everywhere else.

**Review by paulmskim — COMMENTED** [review]
looks good, just one comment

**Review by paulmskim — APPROVED** [review]

**Review by sc0ttkclark — DISMISSED** [review]

**Review by sc0ttkclark — APPROVED** [review]

**Comment by sc0ttkclark** [comment]
@juanfra This looks like it needs snapshot updates for ET+ too https://github.com/the-events-calendar/event-tickets-plus/pull/1209

**Comment by juanfra** [comment]
@sc0ttkclark thanks for the heads up, done ✅

**Pull request merged**